examples/ai18.ipynb

```diff
@@ -63,7 +63,7 @@
     "   * **60 min** &nbsp;*Lunch break*<br><br>\n",
     "   \n",
     "- *Running ML or other analysis or simulation tools*\n",
-    "   * **30 min** &nbsp;[5 - Machine Learning](tutorial/05_Machine_Learning.ipynb): Specifying an ML pipeline using the prepared training data.\n",
+    "   * **30 min** &nbsp;[5 - Machine Learning](tutorial/04_Machine_Learning.ipynb): Specifying an ML pipeline using the prepared training data.\n",
```

Comment:
5 - no longer matches 04_ , and the same below.  Probably better to go ahead and renumber things in this index to match how we'd give the tutorial if we do it again; the old version (should tag it!) can be retrieved if we want to know how we actually gave it.